[2025-05-22T17:00:00.555+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_stream_dag.fetch_weather scheduled__2025-05-22T16:00:00+00:00 [queued]>
[2025-05-22T17:00:00.564+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_stream_dag.fetch_weather scheduled__2025-05-22T16:00:00+00:00 [queued]>
[2025-05-22T17:00:00.565+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-22T17:00:00.579+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_weather> on 2025-05-22 16:00:00+00:00
[2025-05-22T17:00:00.585+0000] {standard_task_runner.py:60} INFO - Started process 1109 to run task
[2025-05-22T17:00:00.589+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_stream_dag', 'fetch_weather', 'scheduled__2025-05-22T16:00:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/weather_stream_dag.py', '--cfg-path', '/tmp/tmpusjnvfuq']
[2025-05-22T17:00:00.591+0000] {standard_task_runner.py:88} INFO - Job 45: Subtask fetch_weather
[2025-05-22T17:00:00.608+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-22T17:00:00.646+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_stream_dag.fetch_weather scheduled__2025-05-22T16:00:00+00:00 [running]> on host dab7539a66ec
[2025-05-22T17:00:00.740+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_stream_dag' AIRFLOW_CTX_TASK_ID='fetch_weather' AIRFLOW_CTX_EXECUTION_DATE='2025-05-22T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-22T16:00:00+00:00'
[2025-05-22T17:00:00.898+0000] {logging_mixin.py:188} INFO - [✓] Données météo sauvegardées : data/raw/weather/weather_2025-05-22T17-00-00.json
[2025-05-22T17:00:00.900+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-22T17:00:00.911+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_stream_dag, task_id=fetch_weather, execution_date=20250522T160000, start_date=20250522T170000, end_date=20250522T170000
[2025-05-22T17:00:00.962+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-22T17:00:00.980+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
